Dockerfiles/agent/entrypoint/60-sysprobe-check.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+if grep -Eq 'enable_tcp_queue_length *: *true' /etc/datadog-agent/system-probe.yaml; then
```

Comment:
hm, looks like this returns true even if it is commented out (` # enable_tcp_queue_length: true`)

Reply:
Good catch !
Fixed !